### ISSUE TITLE
MASI-E001 — Empathy semantic-observation specialist

### ISSUE THREAD

**Issue by JosephJMWalker-MBA** [JosephJMWalker-MBA/masi-research#3]
## Research question
Can a small locally specialized model infer a bounded, evidence-grounded observation vector more reliably than the same untuned local base model, and how does it compare with a larger general-purpose baseline?

## Why this experiment
The Accumulated Distress Care Protocol (ADCP) already separates semantic observations from deterministic downstream care policy. That creates a narrow learned task without asking a model to imitate or own the policy layer.

This is a candidate first test of the MASI proposition that learned specialization can matter more than general capability on a bounded responsibility.

## Required design work before training
- [ ] freeze the target observation schema
- [ ] decide which ADCP dimensions, if any, belong in E001 unchanged
- [ ] define an independent labeling procedure to reduce circularity
- [ ] define provenance/privacy rules for fixtures
- [ ] create negative and boundary controls
- [ ] freeze train / validation / test split logic
- [ ] select one small local base model suitable for the available hardware
- [ ] define an untuned-local baseline
- [ ] define a larger general-purpose baseline
- [ ] define exact scoring metrics
- [ ] distinguish observation-vector accuracy from downstream stage agreement
- [ ] define calibration / abstention measurement if confidence is emitted
- [ ] record resource accounting requirements
- [ ] verify publication rights for any external comparator before publishing performance results

## Initial comparison

```text
B0  deterministic / trivial baseline where meaningful
L1  untuned local base model
S1  same local model after specialization
F1  larger general-purpose model
```

A later experiment may test the specialist inside a composed MASI system. E001 should first establish whether the specialization itself works.

## Success condition
A preregistered, held-out result showing that specialization produces a meaningful improvement over the same local base model on the bounded observation task, with errors and negative controls preserved.

## Important non-claim
Even a strong E001 result would not establish clinical validity, a complete Empathy module, or the overall MASI thesis.

**Comment by JosephJMWalker-MBA** [comment]
## Operator authorization — E001 protocol-binding packet

MASI-E001 is selected as the first named post-WP0 experiment for **design/preregistration only**.

Authorized now:
- bounded prior-art review needed to define the protocol;
- freeze the Empathy-observation responsibility/task;
- decide what ADCP contributes and what must remain outside the observer;
- freeze ontology/state, allowed inputs, prohibited/unavailable information and evidence-span requirements;
- define independent labeling, provenance/privacy and contamination controls;
- operationally define competent controls and control eligibility;
- freeze split logic, metrics/margins, uncertainty/abstention semantics, negative/boundary/adversarial cases and resource budgets;
- define the smallest credible `MASI_TARGET` design constraints from the responsibility outward.

Still unauthorized:
- implementation;
- training/fine-tuning;
- inference/model runs;
- final-result observation;
- runtime expansion;
- K001 execution;
- builder-branch merge.

The completed protocol packet must stop for independent audit before execution.

Carry forward Audit Addendum C:
1. define "competent" controls operationally;
2. if energy is claimed, define its measurement method; otherwise mark it unmeasured;
3. state whether adversarial epistemic-integrity cases are included.

**Comment by JosephJMWalker-MBA** [comment]
## MASI-E001 protocol acceptance record

PR #17 protocol v1.4 has completed independent audit.

**Disposition:** `ACCEPT`  
**Accepted audited head:** `9edd737764815a49929bfe60f104ff7b5a0e40e9`  
**Governing main:** `904bd9bc1c482c5b552b4a2e7075d685d73a1748`

This closes the protocol-design audit loop only. It does **not** authorize data construction, annotation, implementation, training, inference, unlock, or result claims.

The accepted packet remains blocked on **B1–B5** plus a separate explicit operator execution authorization.

The accepted commit is intentionally preserved unchanged; its historical in-file status wording is part of the audited freeze. Current acceptance state is recorded durably on PR #17 and this issue.

**Comment by JosephJMWalker-MBA** [comment]
## Plateau / resume handoff — 2026-09-19

E001 protocol v1.4 is independently accepted and merged, but execution is intentionally deferred.

Current decision:

- preserve E001 as the serious human-validated experiment;
- do **not** rush B1–B5 or recruit the human team yet;
- run a smaller, human-light, compute-heavy experiment first;
- prefer mechanically known ground truth and large unattended perturbation sweeps;
- the Jetson Orin is the preferred candidate substrate for that smaller experiment;
- the pilot may inform **whether/when** E001 is worth executing, but it may not retroactively modify accepted E001 v1.4.

The next MASI task is design only: write a short pre-E001 experiment note that freezes the single uncertainty, mechanical ground truth, minimum comparator set, Orin rationale, and decision consequences before any implementation.

Full resume state is now in `STATUS.md` at commit `39ce0c33b208cd41bae6d86d52b3073fd56877a9`.

No E001 implementation, annotation, training, inference, unlock, or human recruitment is authorized by this handoff.

**Comment by JosephJMWalker-MBA** [comment]
Empathy successor program recorded on 2026-09-22: [docs/EMPATHY_RESEARCH_PROGRAM.md](https://github.com/JosephJMWalker-MBA/masi-research/blob/main/docs/EMPATHY_RESEARCH_PROGRAM.md).

This does **not** amend or expand accepted E001 protocol v1.4. E001 remains the bounded evidence-grounded observation experiment and intentionally dormant under the current plateau.

Successor research questions are tracked separately:
- #20 — E002 stakeholder and perspective representation
- #21 — E003 perspective counterfactuals
- #22 — E004 missing-perspective detection
- #23 — E005 Empathy-governed composition

The immediate resume point remains the smaller human-light pre-E001 experiment in STATUS.md.
